submodules/eks/main.tf

```diff
@@ -153,6 +153,7 @@ locals {
 
   eks_info = {
     cluster = {
+      public_access     = var.eks.public_access
```

Comment:
Should we put a default of `{}` here? If I'm not mistaken, `var.eks` can potentially be just `{}`, and this should then throw an error.

Reply:
Defaults are set in the root vars which pass through to the submodule.
https://github.com/dominodatalab/terraform-aws-eks/blob/8d692ba55c7aa980226060a416d8f282c662d1e6/variables.tf#L79
we do not expect an empty `{}` `eks` from the root module